AIPscan/Data/data.py

```diff
@@ -2,8 +2,10 @@
 
 from datetime import datetime
 
+from AIPscan import db
 from AIPscan.models import AIP, File, FileType, StorageService
 
+VALID_FILE_TYPES = set(item.value for item in FileType)
```

Comment:
I've been finding using the `FileType` enum to be pretty smooth. I'm not quite sure I follow your suggestion around disposition - do you mean to change the name `file_type`/`FileType` or are you envisioning a larger change?

Reply:
Indeed! A name change - mainly because file type means a lot more things to me (especially if we're in the land of formats), and not specifically "'original" or "preservation". but YMMV.

Reply:
Hm, yeah that's fair! I'm not married to "file type" but do think maybe that'd be better changed in a separate PR and all at once?

Reply:
Agreed!